### ISSUE TITLE
add function to recalcuate MLGs

### ISSUE THREAD

**Issue by zkamvar** [grunwaldlab/poppr#19]
If the user has a genclone object, the MLGs are static from when the data is read in to allow for identification downstream. There is no obvious way to reset those MLGs currently. It would be good to provide a small function that would do this.

**Comment by zkamvar** [comment]
This has been added to the `mlg.vector` function, but perhaps it should also be added to `mll()` to allow reset of mlgs based on original, custom, contracted, or any combination of them.

For Example:

```r
mll.reset(myData) <- "original" # reset only the original mlgs, leave the rest alone.
mll.reset(myData) <- "custom" # reset only custom mlgs.
mll.reset(myData) <- "contracted" # reset contracted mlgs (though, I'm not sure how this will work since they are always collapsed starting with the "original")
mll.reset(myData) <- c("original", "custom") # reset both original and custom
mll.reset(myData) <- TRUE # Reset All
```

**Comment by zkamvar** [comment]
After thinking about this, `mll.reset` shouldn't be an assignment method as it irreversibly changes data. This has been addressed.

**Comment by zkamvar** [comment]
fixed in d7db95056342f425003ee79e31a9f2c1f583efd6
